app/controllers/placements/providers/partner_schools/placements_controller.rb

```diff
@@ -21,6 +20,6 @@ def set_partner_school
   end
 
   def placements_scope
-    @partner_school.placements
+    policy_scope(@partner_school.placements, policy_scope_class: Placements::Provider::PlacementPolicy::Scope)
   end
```

Comment:
Hmm. Maybe a convention would be helpful here.

If we were following the other ones commented above ☝🏻 I guess it would mean renaming the method to simply `#placements`.

Or do you think the `_scope` suffix is useful?